molgenis-core-ui/src/main/java/org/molgenis/core/ui/admin/permission/PermissionManagerController.java

```diff
@@ -396,23 +474,23 @@ private Permissions toPluginPermissions(List<Plugin> plugins, Map<ObjectIdentity
 
 	private void removeSidEntityTypePermission(EntityType entityType, Sid sid)
 	{
-		ObjectIdentity objectIdentity = new EntityTypeIdentity(entityType);
+		ObjectIdentity objectIdentity = new RepositoryIdentity(entityType);
 		removePermissionForSid(sid, objectIdentity);
 	}
 
 	private void createSidEntityTypePermission(EntityType entityType, Sid sid,
-			EntityTypePermission entityTypePermission)
+			RepositoryPermission entityTypePermission)
 	{
-		ObjectIdentity objectIdentity = new EntityTypeIdentity(entityType);
+		ObjectIdentity objectIdentity = new RepositoryIdentity(entityType);
 		createSidPermission(sid, objectIdentity,
-				EntityTypePermissionUtils.getCumulativePermission(entityTypePermission));
+				RepositoryPermissionUtils.getCumulativePermission(entityTypePermission));
 	}
 
-	private void createSidPackagePermission(Package package_, Sid sid, EntityTypePermission entityTypePermission)
+	private void createSidPackagePermission(Package package_, Sid sid, RepositoryPermission entityTypePermission)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)